plugins/qcheck-stm/src/config.ml

```diff
@@ -57,29 +93,115 @@ let core_type_is_a_well_formed_sut (core_type : Ppxlib.core_type) =
   let open Ppxlib in
   let open Reserr in
   match core_type.ptyp_desc with
-  | Ptyp_constr (lid, cts) ->
+  | Ptyp_constr (_lid, cts) ->
       let* _ = map acceptable_type_parameter cts in
-      ok (lid, cts)
+      ok ()
   | _ ->
       let str = Fmt.str "%a" Ppxlib_ast.Pprintast.core_type core_type in
       error (Sut_type_not_supported str, Location.none)
 
-let sut_core_type str =
+let value_bindings cfg_uc =
```

Comment:
Maybe rename it `extract_init_sut` or something similar? (If I read it correctly, the current name is a duplicate of the type signature)